src/main/java/seedu/address/logic/commands/room/EditRoomCommand.java

```diff
@@ -95,15 +96,16 @@ private static Room createEditedRoom(Model model, Room roomToEdit,
                                          EditRoomDescriptor editRoomDescriptor) throws CommandException {
         assert (roomToEdit != null);
         int updatedRoomNumber = editRoomDescriptor.getRoomNumber().orElse(roomToEdit.getRoomNumber());
+        TaskList updatedTaskList = editRoomDescriptor.getTaskList().orElse(roomToEdit.getTaskList());
```

Comment:
I think `editroom` command does not allow task to be edited right?

In that case, maybe we don't need to use `editRoomDescriptor.getTaskList()`?

Reply:
yup, you are correct, i removed it